mono/mini/intrinsics.c

```diff
@@ -1831,6 +1831,26 @@ mini_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 			   !strcmp (cmethod_klass_name_space, "System.Runtime.CompilerServices") &&
 			   !strcmp (cmethod_klass_name, "JitHelpers")) {
 		return emit_jit_helpers_intrinsics (cfg, cmethod, fsig, args);
+	}  else if (in_corlib &&
+			   (strcmp (cmethod_klass_name_space, "System") == 0) &&
+			   (strcmp (cmethod_klass_name, "Activator") == 0)) {
+		MonoGenericContext *method_context = mono_method_get_context (cmethod);
+		if (!strcmp (cmethod->name, "CreateInstance") &&
+				fsig->param_count == 0 &&
+				method_context != NULL &&
+				method_context->method_inst->type_argc == 1 &&
+				cmethod->is_inflated &&
+				!mini_method_check_context_used (cfg, cmethod)) {
+			MonoClass *arg0 = mono_class_from_mono_type_internal (method_context->method_inst->type_argv [0]);
+			if (m_class_is_valuetype (arg0) && !mono_class_has_default_constructor (arg0, FALSE)) {
+				MONO_INST_NEW (cfg, ins, MONO_CLASS_IS_SIMD (cfg, arg0) ? OP_XZERO : OP_VZERO);
+				ins->dreg = mono_alloc_dreg (cfg, STACK_VTYPE);
```

Comment:
I bet OP_XZERO won't compile on arm64

Reply:
MONO_CLASS_IS_SIMD will return 0 on ARM

Reply:
https://github.com/mono/mono/blob/d274cfbacdb2aeab0184662997882f3d78991e48/mono/mini/mini.h#L308-L329

Reply:
Yeah should be ok, I thought OP_XZERO usage is wrapped with if-def usually but's not